examples/picorv32_versa5g/Makefile

```diff
@@ -14,16 +14,16 @@ attosoc.json: io_wrapper.v attosoc.v picorv32.v firmware.hex
 	yosys -p "synth_ecp5 -nomux -json $@ -top top" io_wrapper.v attosoc.v picorv32.v
 
 attosoc_out.config: attosoc.json
-	nextpnr-ecp5 --json $< --basecfg ../../misc/basecfgs/empty_lfe5um5g-45f.config --textcfg $@ --um5g-45k
+	nextpnr-ecp5 --json $< --basecfg /usr/share/prjtrellis/misc/basecfgs/empty_lfe5um5g-45f.config --textcfg $@ --um5g-45k
 
 attosoc.bit: attosoc_out.config
 	ecppack $< $@
 
 %.svf: %.bit
-	../../tools/bit_to_svf.py $< $@
+	/usr/share/prjtrellis/tools/bit_to_svf.py $< $@
```

Comment:
Ditto, although an even nicer option here is to not use the script at all and generate the SVF at the same time as the bitstream with `ecppack --svf`